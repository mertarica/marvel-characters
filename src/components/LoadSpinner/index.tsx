import React from "react";
import Container from "react-bootstrap/esm/Container";

const Loader: React.FC = () => {
  return (
    <Container className="c-load-spinner">
      <div className="c-load-spinner__ellipsis">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </Container>
  );
};

export default Loader;
