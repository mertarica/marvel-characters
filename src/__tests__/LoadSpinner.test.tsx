import React from "react";
import renderer from 'react-test-renderer';
import LoadSpinner from "../components/LoadSpinner";

describe("<LoadSpinner />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<LoadSpinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
