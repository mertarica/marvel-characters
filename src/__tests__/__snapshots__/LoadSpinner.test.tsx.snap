// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadSpinner /> renders correctly 1`] = `
<div
  className="c-load-spinner container"
>
  <div
    className="c-load-spinner__ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
